ownCloud/Viewer/ImageDisplayViewController.swift

```diff
@@ -38,12 +54,59 @@ class ImageDisplayViewController : DisplayViewController {
 		}
 	}
 
+	// MARK: - Frame changes
 	override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
 		super.viewWillTransition(to: size, with: coordinator)
 		self.scrollView?.updateScaleForRotation(size: size)
 	}
+
+	// MARK: - Gesture recognizers actions.
+	@objc func tapToZoom() {
+		guard let scrollView = scrollView else {
+			return
+		}
+
+		if scrollView.zoomScale != scrollView.minimumZoomScale {
```

Comment:
@pablocarmu direct comparison of float numbers is evil due to rounding errors

Reply:
I'll change the way I'm comparing those numbers, it is a preferred way of doing this kind of comparisons or simply doing a `<` or `>` works?

Reply:
@pablocarmu I double checked the code, I think here it's fine.